### DISCUSSION TITLE
Setting up the playground locally

### DISCUSSION THREAD

**Discussion by esocarras7** [facebook/lexical#3430] [General]
Hi. I need a brief explanation of how to set up the lexical-playground package in my local environment, or a link to a tutorial on this. Thank you very much.

**Comment by GermanJablo** [comment]
Duplicate of https://github.com/facebook/lexical/discussions/2406

**Comment by esocarras7** [comment]
Thank you for your response. Thank God, I had already managed to get it to
run locally.

Best regards.

**Comment by nerdess** [comment]
A local installation is actually quite simple, just follow the steps here: https://github.com/facebook/lexical#contributing-to-lexical

Side note: It only worked when I used npm. Using yarn I got a "Cannot find module 'vite'"-error.
